### PR TITLE
Add a CloudWatch report command

### PR DESCRIPTION
**Pull request by brikis98**
This PR adds a new `report` command that can be used to write a metric to CloudWatch. If you’re using ec2-snapper in an automated cronjob, you can add alarms around this metric to detect if the cronjob failed.

### CODE REVIEW

**Comment by brikis98** [comment]
@josh-padnick This is ready for review.

File: README.md [hunk 1]
```diff
@@ -121,6 +124,19 @@ You must specify the AWS region (e.g. `--region=us-west-2`) and either the ID (e
...
+Example:
+
+```bash
+ec2-snapper report --region=us-west-2 --metric-name=MyEc2Backup --namespace=MyCustomMetrics --value=1
+```
+
+This command will write a custom metric to the specified region (e.g. `--region=us-west-2`) with the specified name (e.g. `--metric-name=MyEc2Backup`), namespace (e.g. `--namespace=MyCustomMetrics`), and value (e.g. `--value=1`). You can then add monitoring and alerting around this metric.
+
+For example, let's say you use a cronjob to run ec2-snapper once per night, and if the job completes successfully, you fire the metric as shown in the example above. In that case, you could create a CloudWatch alarm that goes off if the value of the `MyEc2Backup` metric is less than 1 over a 24 hour period. That way, you'd be notified if your cronjob failed for any reason.
+
```
Comment (josh-padnick) [inline README.md:139]: I like the `ec2-snapper report` subcommand interface. Nice choice.

**Comment by josh-padnick** [comment]
Looks great. My only comment is we should further clarify how the CloudWatch alarm should be setup.  For example, you'll want to setup an alarm on `INSUFFICIENT_DATA`, not just `0`.  Otherwise, ready to merge.

**Comment by brikis98** [comment]
I updated the docs to mention INSUFFICIENT_DATA state.

**Pull request merged**